audit/projects/kubernetes-public/iam.json

```diff
@@ -4,7 +4,13 @@
       "members": [
         "group:k8s-infra-cluster-admins@kubernetes.io"
       ],
-      "role": "projects/kubernetes-public/roles/ServiceAccountLister"
+      "role": "organizations/758905017065/roles/iam.serviceAccountLister"
+    },
+    {
+      "members": [
+        "group:k8s-infra-rbac-slack-infra@kubernetes.io"
+      ],
+      "role": "organizations/758905017065/roles/secretmanager.secretLister"
```

Comment:
This was me manually trialing https://github.com/kubernetes/k8s.io/issues/1731#issuecomment-789959343 to help land https://github.com/kubernetes/k8s.io/pull/1696

It did not solve the problem, so I'll manually remove this binding

Reply:
```
$ gcloud projects remove-iam-policy-binding kubernetes-public --member="group:k8s-infra-rbac-slack-infra@kubernetes.io" --role="organizations/758905017065/roles/secretmanager.secretLister"
```